### ISSUE TITLE
[Bug] Cron module fails to create cronjob when gather_facts: True

### ISSUE THREAD

**Issue by adm-github** [ansible/ansible#8695]
##### Environment:

From: mac os x
Remote: Centos 5
##### Summary:

The cron module fails with "a duplicate parameter was found in the argument string ()" but succeeds if you set gather_facts: False
##### Steps To Reproduce:

$ ansible-playbook  -i myhosts myplay.yml
[...]
GATHERING FACTS ***************************************************************
ok: [hostname]

TASK: [cron name = "ajobname" hour = "4" job = "test.sh" user = "myuser"] ***
fatal: [hostname] => a duplicate parameter was found in the argument string ()

FATAL: all hosts have already failed -- aborting
hostname              : ok=1    changed=0    unreachable=1    failed=0

-- myplay.yml --
- hosts: hostgroup
  tasks:
  - action: cron name="ajobname"
          hour="4"
          job="test.sh"
          user="myuser"
##### Expected Results:

all green
##### Actual Results:

TASK: [cron name = "ajobname" hour = "4" job = "test.sh" user = "myuser"] ***
fatal: [hostname] => a duplicate parameter was found in the argument string ()

**Comment by mpdehaan** [comment]
Hmm, this is probably not related to the gather_facts setting, but will investigate.

**Comment by jimi-c** [comment]
@adm-github - I just tested this against a CentOS 5.9 system (though my source is linux), and was not able to reproduce the error using 1.7.1 or the current devel branch:

<pre>
# ansible-playbook --version
ansible-playbook 1.7.1 (release1.7.1 268e72318f) last updated 2014/08/14 15:50:43 (GMT -500)
# ansible-playbook -vv 8695.yml 
PLAY [centos59] *************************************************************** 
GATHERING FACTS *************************************************************** 
<192.168.122.121> REMOTE_MODULE setup
ok: [192.168.122.121]
TASK: [cron name="ajobname" hour="4" job="test.sh" user="root"] *************** 
<192.168.122.121> REMOTE_MODULE cron name="ajobname" hour="4" job="test.sh" user="root"
ok: [192.168.122.121] => {"changed": false, "jobs": ["ajobname"]}
PLAY RECAP ******************************************************************** 
192.168.122.121            : ok=2    changed=0    unreachable=0    failed=0   
</pre>

It is possible this is related to the source being OSX though, if you can provide any further details regarding your configuration, like how you installed Ansible, if you've upgraded recently, etc.?

**Comment by adm-github** [comment]
For some reasons I reverted the change to gather_facts and now it works. It must be something in my env. Closing for now. Info requested:
Installed with git clone.
$ git status
On branch devel
Your branch is up-to-date with 'origin/devel'.

nothing to commit, working directory clean

$ ansible --version
ansible 1.7.1

Darwin $hostname 13.3.0 Darwin Kernel Version 13.3.0: Tue Jun  3 21:27:35 PDT 2014; root:xnu-2422.110.17~1/RELEASE_X86_64 x86_64
